.github/workflows/long-test.yml

```
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

```suggestion
    - cron: "0 0 * * 0-4"
```

Probably. Run on Sunday night, so we have a status Monday morning, and the rest of the week. Friday and Saturday night runs are not useful.

---

Isn't 1-5 a correct schedule to run Mon-Fri?

---

Sure, that's fine too. I guess the expectation in this case is that on the Monday, you have the build from the previous Friday night. My proposal (Sunday to Thursday) is a little more conservative: if stuff gets checked in during the weekend, up to Sunday midnight, then you get it in the build you see on Monday morning.

---

Are we both on the same page assuming that 0 0 * * 1-5 means
- 12:00AM Mon (nightly after Sunday, fresh on Monday morning)
- 12:00AM Tue (nightly after Monday, fresh on Tuesday morning)
- ...
- 12:00AM Fri (nightly after Thursday, fresh on Friday morning)

?

---

@maxtropets no, I was wrong, and clearly you are right. 1-5.